cmd/csi-driver/csi-driver.go

```diff
@@ -71,6 +73,7 @@ func init() {
 	RootCmd.PersistentFlags().StringVarP(&dbServer, "dbserver", "s", "", "Database server for the CSI driver")
 	RootCmd.PersistentFlags().StringVarP(&dbPort, "dbport", "p", etcd.DefaultPort, "Database server port for the CSI driver")
 	RootCmd.PersistentFlags().BoolP("node-service", "", false, "CSI node-plugin")
+	RootCmd.PersistentFlags().Int64Var(&scrubberInterval, "scrubber-interval", csiScrubberInterval, "Scrubber interval in seconds for the CSI driver")
```

Comment:
Same comment here: Maybe calling it inlineVolumeScrubberInterval would make this more obvious.